projects/plugins/jetpack/modules/subscriptions/newsletter-widget/src/components/newsletter-widget.tsx

```diff
@@ -115,9 +115,9 @@ export const NewsletterWidget = ( {
 						{
 							link: DashboardLink(
 								isWpcomSite,
-								getRedirectUrl(
-									buildJPRedirectSource( 'learn/courses/newsletters-101/wordpress-com-newsletter' )
-								),
```

Comment:
@jeherve was there any special purpose of using these helper functions here instead of giving the link directly?

I have following understanding of these helper functions, it would be great if you can tell me if it is correct 🙂 

- `buildJPRedirectSource` is needed to determine whether we should redirect the users to Dotcom page or Jetpack Cloud page.
- `getRedirectUrl` to pass the user the Jetpack redirect service that may have some tracking or logging.

Reply:
You're correct, yes. We want to redirect self-hosted site owners to Jetpack cloud, but not WordPress.com site owners, and `getRedirectUrl` adds some tracking.